libvast/src/format/pcap.cpp

```diff
@@ -28,15 +27,15 @@
 #  include <caf/settings.hpp>
 
 #  include <cstddef>
+#  include <pcap.h>
 #  include <string>
 #  include <thread>
 #  include <utility>
 
 #  include <netinet/in.h>
 
-namespace vast {
```

Comment:
Nit: I know your local Clang format 11 collapsed these nested namespace, but it may be cleaner to just apply this in a one-shot-go for all files in a separate PR. This has the benefit of being consistent going forward rather than this sort of thing coming up as small diffs in many files over time.  We can ignore the Git revision for the formatting easily to avoid `git blame` and friends for just formatting revisions in the Git history.

Reply:
This was actually clang-tidy, not clang-format. The clang-format issue was totally separate.

> We can ignore the Git revision for the formatting easily to avoid git blame and friends for just formatting revisions in the Git history.

It's a matter of policy. Right now we only apply clang-format to the hunks that were changed from the PR merge base.